recipes/libgpiod/all/conanfile.py

```diff
@@ -24,59 +30,65 @@ class LibgpiodConan(ConanFile):
         "enable_bindings_cxx": False,
         "enable_tools": False,
     }
-    
-    _autotools = None
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def validate(self):
         if self.settings.os != "Linux":
             raise ConanInvalidConfiguration("libgpiod supports only Linux")
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.rm_safe("fPIC")
+
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
         if not self.options.enable_bindings_cxx:
-            del self.settings.compiler.libcxx
-            del self.settings.compiler.cppstd
+            self.settings.rm_safe("compiler.libcxx")
+            self.settings.rm_safe("compiler.cppstd")
+
+    def requirements(self):
+        self.requires("linux-headers-generic/5.13.9")
```

Comment:
See this: https://github.com/conan-io/conan-center-index/pull/18171#issuecomment-1616038374